src/WebJobs.Extensions.DurableTask/LinuxLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    public class LinuxLogger : ILogger
+    {
+        private readonly bool inConsumption;
+        private readonly string consolePrefix = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+        private readonly string loggingPath = "/var/log/functionsLogs/durableevents.log";
+
+        public LinuxLogger(bool inConsumption)
+        {
+            this.inConsumption = inConsumption;
+        }
+
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            // TODO: I this we can leave this unimplemented
+            throw new NotImplementedException();
+        }
+
+        public bool IsEnabled(LogLevel logLevel)
+        {
+            // TODO: assuming these logs should never be disabled
+            return true;
+        }
+
+        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        {
+            string jsonString = ""; // assume that, somehow, this becomes a string-representation of a JSON-representation of an ETW event....
+            // @Chris, any ideas on how to set the right value here? I suppose we'll need to be listening to ETW events as well and transform them into a JSON-string here.
```

Comment:
I think we might be able to do something like this:
```csharp
JObject json = JObject.FromObject(state); // the state object is basically a dictionary of key-value log pairs
// TODO: maybe add some fields to the JSON object?
string jsonString = json.ToString(Formatting.None);
```

Reply:
Great, that makes sense. I have yet to see how we'd call `logger.Log(state)` with a `state` that contains the ETW data, but we can worry about that when we get to it :)

Reply:
So that's a good point, actually. The DTFx log statements all pass in a `state` object that implements `IReadOnlyDictionary<string, object>` (see [here](https://github.com/Azure/durabletask/blob/ce153b3f9fe7ad053d7b6cc02300707079521a23/src/DurableTask.Core/Logging/StructuredLogEvent.cs#L29)). However, WebJobs.Extensions.DurableTask does *not* write logs in this format. It's possible that the implementation of `ILogger.LogInformation(...)` etc. creates a usable `state` object that we can serialize as-is, but we'd need to test it out to be sure.

However, we have to also be aware that our customer `ILogger` might also capture logs written by components we don't care about, like Functions runtime logs or customer logs. We don't want any of those logs to show up in our Durable-specific Kusto tables so we'll need to do some kind of filtering here. For example, I can imagine we might do something like this:

```csharp
if (state?.GetType().FullName.Contains("DurableTask") != true)
{
   // ignore log statements that do not come from Durable sources
   return;
}
```